apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -362,15 +368,16 @@ export const submitChatContents = createAsyncThunk(
         .logError('Error in aichat completion request', error as Error);
 
       thunkAPI.dispatch(clearChatMessagePending());
-      thunkAPI.dispatch(addChatMessage({...newMessage, status: Status.ERROR}));
+      const erroredUserChatMessage = {...newMessage, status: Status.ERROR};
+      thunkAPI.dispatch(addChatEvent(erroredUserChatMessage));
```

Comment:
nit: any reason to not keep this inlined?

Reply:
Oh yeah - I think I iterated on this a bit. Can keep inlined.